stdlib/3/distutils/dist.pyi

```diff
@@ -1,8 +1,56 @@
 from distutils.cmd import Command
-from typing import Any, Dict, Iterable, Mapping, Optional, Tuple, Type
+from typing import IO, Any, AnyStr, Dict, Iterable, List, Mapping, Optional, Tuple, Type
+
+class DistributionMetadata:
+    def __init__(self, path: Optional[str] = ...): ...
```

Comment:
As `path` gets passed to `open()`, this can accept `Union[int, AnyPath]`. (`AnyPath` can be imported from `_typeshed`.)